src/data/queries/index.ts

```diff
@@ -51,17 +52,20 @@ export const QUERIES_MAP = {
   'node--story_listing': StoryListing,
   'node--q_a': QuestionAnswer,
   'node--event': Event,
+  'node--vet_center_facility_health_servi': HealthServices,
   'node--event--teaser': EventTeaser,
   'node--event_listing': EventListing,
   'node--person_profile': PersonProfile,
   'node--landing_page': BenefitsHub, // "Benefits Hub Landing Page"
   'node--support_service': SupportServices,
   'node--support_resources_detail_page': ResourcesSupport,
+  'node--vet_center': VetCenter,
+
   // Paragraphs
   'paragraph--alert': Alert,
   'paragraph--alert_single': AlertSingle,
-  'paragraph--audience_topics': AudienceTopics,
   'paragraph--basic_accordion': Accordion,
+  'paragraph--audience_topics': AudienceTopics,
```

Comment:
keep this above basic_accordion, more alphabetical based on paragraph name, not component